Tests/BrowserServicesKitTests/Utils/JsonTestDataLoader.swift

```diff
@@ -27,9 +27,26 @@ enum FileError: Error {
 final class FileLoader {
 
     func load(filePath: String, fromBundle bundle: Bundle) throws -> Data {
+        
+        guard let resourceUrl = bundle.resourceURL else { throw FileError.unknownFile }
+        
+        let url = resourceUrl.appendingPathComponent(filePath)
+        
+        let finalURL: URL
+        if FileManager.default.fileExists(atPath: url.path) {
+            finalURL = url
+        } else {
+            // Woraround for resource bundle having a different structure when runnign tests from command line.
```

Comment:
it's a minor but make sure to turn on check spelling under: Edit/Format/Spelling and Grammar/Check Spelling While Typing, so you will avoid any typos in the comments and variable names